src/libs/ajax/DatasetBuilder.ts

```diff
@@ -1,59 +1,27 @@
 // Types that can be used to create a criteria.
 import _ from 'lodash/fp';
-
-export interface DomainOption {
-  kind: 'domain';
-  id: number;
-  category: string;
-  conceptCount: number;
-  participantCount: number;
-  root: Concept;
-}
-
-export interface ProgramDataOption {
-  kind: 'range' | 'list';
-  id: number;
-  name: string;
-}
-
-export interface ProgramDataRangeOption extends ProgramDataOption {
-  kind: 'range';
-  min: number;
-  max: number;
-}
-
-export interface ProgramDataListValue {
-  id: number;
-  name: string;
-}
-
-export interface ProgramDataListOption extends ProgramDataOption {
-  kind: 'list';
-  values: ProgramDataListValue[];
-}
-
-export interface DatasetResponse {
-  name: string;
-  id: string;
-  description: string;
-  programDataOptions: (ProgramDataRangeOption | ProgramDataListOption)[];
-  domainOptions: DomainOption[];
-  learnMoreLink: string;
-  accessLevel: AccessLevel;
-  featureValueGroups: FeatureValueGroup[];
-}
+import { Ajax } from 'src/libs/ajax';
+import {
+  datasetIncludeTypes,
+  DatasetModel,
+  ProgramDataListOption,
+  ProgramDataListValue,
+  ProgramDataRangeOption,
+  SnapshotBuilderConcept,
+  SnapshotBuilderDomainOption,
```

Comment:
Would make sense to alias some of these names so they're shorter here? Or does that make it harder to tell what's going on?